app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/NewTransferNavHost.kt

```diff
@@ -43,12 +46,16 @@ fun NewTransferNavHost(
     NavHost(navController, NewTransferNavigation.startDestination) {
         composable<ImportFilesDestination> {
             ImportFilesScreen(
+                importFilesViewModel = hiltViewModel<ImportFilesViewModel>(it),
```

Comment:
Does it still make sense to keep the default value inside ImportFilesScreen() definition if we're always specifying it here instead now?